src/main/java/org/myrobotlab/service/Arduino.java

```diff
@@ -365,20 +365,20 @@ public void attachI2CControl(I2CControl control) {
     // the i2c bus here and in MrlComm
     // This will only handle the creation of i2cBus.
     if (i2cBus == null) {
-      i2cBus = new I2CBus(String.format("I2CBus%s", control.getDeviceBus()));
-      i2cBusAttach(i2cBus, Integer.parseInt(control.getDeviceBus()));
+      i2cBus = new I2CBus(String.format("I2CBus%s", control.getBus()));

```

Comment:
What's with the changes from getDeviceBus/Address to getBus/Address? Is there any difference between them or just a naming change?

Reply:
naming change only ...